<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpsExportWidget</class>
 <widget class="QWidget" name="GpsExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPS Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="inputFileLabel">
     <property name="text">
      <string>1. Select input file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="inputFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="inputFileButton">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="cameraTypeLabel">
       <property name="text">
        <string>Camera Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cameraTypeLine">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cameraSupportLine">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>2. Output Format</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="outputFormatComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="outputFileLabel">
     <property name="text">
      <string>3. Output File</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="outputFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="outputFileButton">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="exportButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
